spec/factories/judges.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :judge do
+    casa_org { create(:casa_org) }
```

Comment:
This is fine, but for future simplicity, you can just write it like:

```
FactoryBot.define do
  factory :judge do
    casa_org
    name { Faker::Name.name }
    active { true }
  end
end
```
And it will automatically create the `casa_org` association.